src/main/java/net/starschema/clouddb/jdbc/BQConnection.java

```diff
@@ -155,6 +155,7 @@ public BQConnection(String url, Properties loginProp, HttpTransport httpTranspor
     String userKey = caseInsensitiveProps.getProperty("password");
     String userPath = caseInsensitiveProps.getProperty("path");
 
+    String targetServiceAccount = caseInsensitiveProps.getProperty("targetserviceaccount");
```

Comment:
should be this targetaccount? or targetserviceaccount? Looking at the properties file I added for a secret -- I think that uses targetaccount instead of targetserviceaccount.